Answers.md

```diff
@@ -0,0 +1,13 @@
+1. This allows you to write cleaner code and keep your css
```

Comment:
Your answers are on par, if you want to go the extra step, you can describe it in more detail.